src/extensions/registries/status-bar-registry.ts

```diff
@@ -3,8 +3,12 @@
 import React from "react";
 import { BaseRegistry } from "./base-registry";
 
+export interface StatusBarComponents {
+  Icon?: React.ComponentType<any>;
```

Comment:
Why `Icon`? Why not `Item`?

Reply:
I guess I was caught up with the specific example I was working with. Good catch.